scripts/kconfig/kconfigfunctions.py

```diff
@@ -88,10 +88,40 @@ def dt_chosen_path(kconf, _, chosen):
     return node.path if node else ""
 
 
+def dt_node_enabled(kconf, name, node):
+    """
+    This function is used to test if a node is enabled or not. If it's
+    called as "dt_path_enabled", it works with either aliases or
+    paths. If it is called as "dt_alias_enabled", it works only with
+    aliases.
```

Comment:
Can we add to the comment about the 'node' argument being either the path or alias.